### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by cbensimon** [gradio-app/gradio#228]
Hi Gradio team,

It seems to me that the python process uses 100% CPU on idle (one full CPU core)

Steps to reproduce :

app.py
```python
import gradio as gr

def greet(name):
  return "Hello " + name + "!"

iface = gr.Interface(fn=greet, inputs="text", outputs="text")
iface.launch()
```

Run the gradio app and see CPU usage
``` bash
docker run -d --name gradio -v $(pwd)/app.py:/app.py python:3.8.9 /bin/bash -c "pip install gradio && python /app.py"
docker stats gradio --no-stream --format "{{.CPUPerc}}"
```

Output :
```
100.00%
```

**Comment by aliabid94** [comment]
Thanks for pointing this out @cbensimon, should be fixed now as of 2.0.9

**Comment by cbensimon** [comment]
It seems to be fixed with 2.0.9 on my side ! Thanks
